src/routes/Profile.tsx

```diff
@@ -42,11 +42,21 @@ export function Profile() {
 
     const [copy, copied] = useCopy({ copiedTimeout: 1000 });
 
+    const profileDeleted = createMemo(() => {
+        // return true if profile.deleted is true or "true"
+        // return false if profile.deleted is false or "false"
+        if (profile().deleted === true || profile().deleted === "true") {
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
this can just be `(profile().deleted === true || profile().deleted === "true")` no need for the if statement